pkg/metrics/labels.go

```diff
@@ -11,6 +11,11 @@ import (
 	"github.com/prometheus/client_golang/prometheus"
 )
 
+const (
+	LabelPolicy          = "policy"
+	LabelPolicyNamespace = "policy_namespace"
```

Comment:
We can also define whole `UnconstrainedLabel` structs here, like we do for constrained labels OpCodeLabel and EventTypeLabel. Not blocking though.

Reply:
ok